core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/CompactEntityComponent.scala

```diff
@@ -181,6 +181,20 @@ class CompactEntityQuery(driverComponent: DriverComponent) extends RawSqlQuery w
       query.as[CompactEntityVersionRecord]
     }
 
+  /**
+   * Get all entities of a given type in a workspace.
+   *
+   * `execution plan: index range scan; using where. Index: idx_entity_type_name`
+   */
+  def listEntities(workspaceId: UUID, entityType: String): ReadAction[Seq[CompactEntityRecord]] =
+    multipleEntityQuery(workspaceId, entityType).as[CompactEntityRecord]
+
+  private def multipleEntityQuery(workspaceId: UUID, entityType: String) =
```

Comment:
I would inline this... there's no need to have an extra function if it is not reused or does not improve readability.